client/src/partials/cash/cash.html

```diff
@@ -50,7 +50,7 @@
                 <bh-currency-select
                   currency-id="CashCtrl.payment.currency_id"
                   validation-trigger="CashVoucherForm.$submitted"
-                  on-change="CashCtrl.digestExchangeRate()"
+                  on-change="CashCtrl.digestExchangeRate(currency)"
                   disable-ids="CashCtrl.disabledCurrencyIds"
                   cashbox-id="CashCtrl.cashbox.id"
```

Comment:
Actually, the only dependency here is for an `<a href>`  to link to the correct `/cashboxes/:id/update` URL.  Otherwise, the component doesn't depend on it and it is optional.